pkg/virtualtable/virtualtable.go

```diff
@@ -589,14 +591,21 @@ func ExpandAndReturnIndexNames(indexNameIn string, orgid uint64, isElastic bool)
 			return []string{}
 		}
 		for indexName := range indexNames {
+			if isIndexExcluded(indexName) {
+				continue
+			}
 			if !isElastic && strings.Contains(indexName, ".kibana") {
 				continue
 			}
+
 			finalResultsMap[indexName] = true
 		}
 	} else {
 		indexNames := strings.Split(indexNameIn, ",")
 		for _, indexName := range indexNames {
+			if isIndexExcluded(indexName) {
+				continue
+			}
 			if strings.Contains(indexName, "*") {
 				regexStr := "^" + strings.ReplaceAll(indexName, "*", `.*`) + "$"
```

Comment:
move the above if loop `if isIndexExcluded(indexName) {..} `inside the loop `if strings.Contains(indexName, "*")`